### PR TITLE
Bug 1215086 - Resolve the two records have the same URL.

### PR DESCRIPTION
**Pull request by weilonge**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=07f7d01b105ddc4a56a263a918aa7c721b81ab83

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6399fab9acbe8a4a386c82a3ed951c9e64d838a3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=38ddbabc8b6ad4d0caafdc951c5d64bfe19283f7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ac119f6df2e067411a0181967493b7a55c4f6257

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=04739f6cff81cbd6ab8f8578bd4c5e05284db36c

File: build/config/common-settings.json [hunk 1]
```diff
@@ -235,7 +235,7 @@
    "support.callsupport1.href": "",
    "support.callsupport2.title": "",
    "support.callsupport2.href": "",
-   "sync.collections.history.enabled": true,
+   "sync.collections.history.enabled": false,
```
Comment (ferjm) [inline build/config/common-settings.json]: Do not change this, please

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6d9762ff5db24ce697e693b3dc4a72475395f9ba

**Pull request merged**